sedas_pyapi/sedas_api.py

```diff
@@ -143,10 +156,11 @@ def search_optical(self, _wkt: str, _start_date: str, _end_date: str, **_filters
         :param _wkt: wkt formatted aoi
         :param _start_date: start date of search in ISO8601 format
         :param _end_date: end date of search in ISO8601 format
+        :param _satellite_name: name of the satellite to search
         :param _filters: filter search on
         :return: list of search results
         """
-        return self.search(_wkt, _start_date, _end_date, 'Optical', **_filters)
+        return self.search(_wkt, _start_date, _end_date, 'Optical', _satellite_name=_satellite_name, **_filters)
```

Comment:
You're defaulting _satellite_name to "" in the definition, and then forcing it to _satellite_name here? Why not just pass it straight through?